phpBB/adm/style/acp_main.html

```diff
@@ -93,64 +93,98 @@ <h3>{L_ERROR_MBSTRING_HTTP_OUTPUT}</h3>
 	<!-- ENDIF -->
 
 	<!-- EVENT acp_main_notice_after -->
-
-	<table class="table1 two-columns no-header" data-no-responsive-header="true">
-		<caption>{L_FORUM_STATS}</caption>
-		<col class="col1" /><col class="col2" /><col class="col1" /><col class="col2" />
+	
+	<div id = "lside">
```

Comment:
remove spaces before and after `=` and convert to class ids are never to be used